### ISSUE TITLE
[Regression] `doNotPrerender` does not allow `false` or `undefined` value

### ISSUE THREAD

**Issue by spacedawwwg** [brillout/vite-plugin-ssr#382]
### Description

Up to now we have been using a method, that returns a boolean, to determine whether a page should be pre-rendered. This allows to conditionally decide whether a page should be pre-rendered based on environment.

```
export const doNotPrerender = !canPrerenderType('brandPage');
``` 

as of 0.4.12, builds now fail with the below error:

```
Error: [vite-plugin-ssr@0.4.12][Wrong Usage] /src/pages/UcoBrand/UcoBrand.page.server.ts has `export { doNotPrerender }` with a forbidden value. The value of `doNotPrerender` is only allowed to be `true`.
```

### Error Stack
```
/Path/to/project/node_modules/@brillout/libassert/dist/cjs/newError.js:10
        err = new Error(errorMessage);
              ^

Error: [vite-plugin-ssr@0.4.12][Wrong Usage] /src/pages/UcoBrand/UcoBrand.page.server.ts has `export { doNotPrerender }` with a forbidden value. The value of `doNotPrerender` is only allowed to be `true`.
    at null.<anonymous> (/Path/to/project/node_modules/vite-plugin-ssr/dist/cjs/shared/getPageFiles/assertExports.js:13:21)
    at Array.forEach (<anonymous>)
    at null.assertExportValues (/Path/to/project/node_modules/vite-plugin-ssr/dist/cjs/shared/getPageFiles/assertExports.js:7:17)
    at Object.pageFile.loadFile (/Path/to/project/node_modules/vite-plugin-ssr/dist/cjs/shared/getPageFiles/parseGlobResults.js:27:37)
    at null.<anonymous> (/Path/to/project/node_modules/vite-plugin-ssr/dist/cjs/node/prerender.js:125:9)
```

**Comment by brillout** [comment]
Makes sense. Let me fix this.

**Comment by brillout** [comment]
I'll fix this later today/tomorrow.

**Comment by brillout** [comment]
I didn't test the fix; let me know if it doesn't work.

I'll release the fix later today / tomorrow.

**Comment by spacedawwwg** [comment]
@brillout I don't think 0.4.13 has fixed this, in fact I think it may have have another issue.

I updated to 0.4.13 and tried a build with my `export const doNotPrerender = !canPrerenderType('takeAction');` conditional function - but it fails.

So I got rid of of the function and just set `export const doNotPrerender = true` - but unfortunately that fails, too.

<img width="534" alt="image" src="https://user-images.githubusercontent.com/1691867/181212263-a951001c-a678-4de9-a582-1748b9225d47.png">
<img width="227" alt="image" src="https://user-images.githubusercontent.com/1691867/181212570-cc304096-2a4a-4ee7-99a7-0bfd6b4b4506.png">
